"error" = "Erro";
"failedToProccessTheResultsError" = "Falha ao processar os resultados do Github.";
"apiRequestError" = "Falha ao receber os dados do Github, verifique sua conex√£o e tente novamente.";
"unexpectedError" = "Um erro inesperado ocorreu.";
"pullToRefresh" = "Puxe para atualizar";
"xRatingLimitError" = "O limite de requisi√ß√µes no Github foi alcan√ßado, aguarde %i segundos e tente novamente.";
"stars" = "Estrelas";
"star" = "Estrela";
"confirm" = "Ok";
"unexpectedNetworkError" = "Um erro inesperado de rede ocorreu, por favor tente novamente.";
